x-pack/plugins/fleet/server/services/package_policies/experimental_datastream_features.test.ts

```diff
@@ -100,7 +100,7 @@ describe('experimental_datastream_features', () => {
               mappings: {
                 _source: {
                   mode: 'stored',
-                },
+                } as any, // TODO: Fix type failure introduced by elasticsearch-js 8.6.0-canary.3
```

Comment:
@nchaulet Since the experimental feature has been removed does this test still make sense?

Reply:
The experimental feature should still be here, where do you see this has been removed?

Reply:
The client no longer supports the types which means these types were probably removed upstream from Elasticsearch. It might mean they just decided to no longer include experimental types.

Reply:
It seems to me that we still support it https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-source-field.html#synthetic-source

Reply:
@rudolf good catch here the _souce.mode:stored is no more supported I have a PR to fix that here https://github.com/elastic/kibana/pull/149209